app/assets/javascripts/navbar/navbarController.js

```diff
@@ -0,0 +1,14 @@
+angular.module('ocWebGui.navbar', ['ui.router', 'ocWebGui.login', 'ocWebGui.shared.fullscreen'])
+  .controller('NavbarController', function ($scope, $http, User, MyFullscreen) {
+    var vm = this;
+    
+    vm.isFullscreen = false;
+    vm.goFullscreen = function () {
+      MyFullscreen.goFullScreen();
+      vm.isFullscreen = !vm.isFullscreen;
```

Comment:
`vm.isFullscreen` variable doesn't reflect current fullscreen state because it can change without this button for example by pressing ESC-key. To fix this use `FBFullscreen.change` event or `fullscreen` directive provided by angular-fullscreen.